pkg/neorpc/filters.go

```diff
@@ -11,7 +11,7 @@ type (
 	// since/till the specified index inclusively). nil value treated as missing
 	// filter.
 	BlockFilter struct {
-		Primary *int    `json:"primary,omitempty"`
+		Primary *byte   `json:"primary,omitempty"`
```

Comment:
What happens on unmarshalling a JSON with `"primary": 100500`?

Reply:
neorpc.Error{Code:-32602, Message:"Invalid params", Data:"json: cannot unmarshal number 100500 into Go struct field BlockFilter.primary of type uint8"

Reply:
I think it's OK, isn't it? That was the whole point of https://github.com/nspcc-dev/neo-go/issues/3241#issue-2015972066. @roman-khimov, don't you think so?

Reply:
OK, invalid is invalid.